feat: rewrite js rpg project (#46515)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a3a488b24fb32b91155d56.md

---
id: 62a3a488b24fb32b91155d56
title: Step 19
challengeType: 0
dashedName: step-19
---

# --description--

> Variables can be <dfn>assigned</dfn> a value. When you do this at the same time that you declare it, this is called <dfn>initialization</dfn>. For example:

Reviewer: ```suggestion
Variables can be <dfn>assigned</dfn> a value. When you do this while you declare it, this is called <dfn>initialization</dfn>. For example:
```
